SourceX/storm/storm.cpp

```diff
@@ -588,6 +588,8 @@ void SVidPlayBegin(char *filename, int a2, int a3, int a4, int a5, int flags, HA
 			ErrSdl();
 		}
 	}
+#elif SDL1_VIDEO_MODE_BPP == 8

```

Comment:
@MBeijer are the changes in this file still needed?

Reply:
Yes

Reply:
These changes are there to set the proper resolution while playing video on 8bit, instead of doing scaling, which takes too much CPU from 68k and possibly other older targets requiring 8bit screen mode.

Reply:
It could perhaps be done prettier, but it does its job as is.

Reply:
Ok, thanks. I couldn't remember if that was already handled some where else.